csds-client/client/client_test.go

```diff
@@ -0,0 +1,171 @@
+package client
+
+import (
+	envoy_service_status_v2 "github.com/envoyproxy/go-control-plane/envoy/service/status/v2"
+
+	"bytes"
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/encoding/prototext"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+	"testing"
+)
+
+// test parsing -request_file to nodematcher
+func TestParseNodeMatcherWithFile(t *testing.T) {
+	c := Client{
+		info: Flag{
+			requestFile: "./test_request.yaml",
+		},
+	}
+	err := c.parseNodeMatcher()
```

Comment:
Error check can be inlined 

if err := fn(); err != nil {}

Reply:
Solved.